docker-compose.emily.yml

```diff
@@ -0,0 +1,70 @@
+services:
+
+  # Hosts the DynamoDB Tables.
+  dynamodb:
+    image: "amazon/dynamodb-local:latest"
+    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath ./data"
+    ports:
+      - "8000:8000"
+    volumes:
+      - "./devenv/dynamodb/data:/home/dynamodblocal/data"
```

Comment:
Oh I noticed that a database file shows up here as `devenv/dynamodb/data/shared-local-instance.db`. I think we should `.gitignore` it.

Reply:
Done.